.github/workflows/build_release.yml

```diff
@@ -7,6 +7,10 @@ on:
     branches:
       - main
   pull_request:
+    paths:
+      - 'lq1/**'
+      - 'qsrc/**'
+      - 'texture-wads/**'
```

Comment:
@lavenderdotpet Do we use the scripts in the `bin/` path for any reason?

Reply:
i mainly have it there to keep track of how to do some thing in qpakman

Reply:
Thanks! We should probably remove it then to reduce repository clutter, if you could keep it elsewhere that would be very much appreciated. Just wanted to make sure we didn't need to include it into this path.